meadow-client/webpack.config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     module: {
         rules: [
             { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url-loader?limit=10000&mimetype=application/font-woff" },
-            { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "file-loader" },
+            { test: /fontawesome-.*\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "file-loader" },
```

Comment:
next time someone adds even more regexness they should probably add a comment, it's getting to the "Perl point"

Reply:
Funny and fair.